### DISCUSSION TITLE
Change name of OD stat from "Accuracy" to "Strictness"

### DISCUSSION THREAD

**Discussion by vatei** [ppy/osu#29557] [Ideas]
In an effort to make the Overall Difficulty beatmap stat less misleading, it got renamed to Accuracy in Lazer, which is even more confusing.
Since OD represents how strict the timing windows are, how about renaming it to "Strictness" instead? Gives a better idea and is pretty simple to understand and remember.
Thanks for considering my simple proposal :)

**Comment by Tenexxt** [comment]
Both Accuracy is a missleading name techinically since OD also changes the difficulty of spinners, tho Strictness makes more sense, it just doesn't feel right (atleast for me).
I have no idea what other name it could be, but id still look for a better name.

**Reply by vatei** [reply]
I had "Judgment sensitivity" but that obviously doesn't convey the spinner changes
Maybe something that means nothing like "Discipline"? (or "Overall discipline" to keep the OD acronym lol)
Idk though, Strictness seems the most appropriate, even considering spinners I guess
